applications/crossbar/src/api_util.erl

```diff
@@ -622,7 +622,7 @@ is_cb_module(Context, Elem) ->
 -spec is_cb_module(kz_term:ne_binaries()) -> boolean().
 is_cb_module([]) -> 'false';
 is_cb_module([Module|Modules]) ->
-    kz_module:is_exported(Module, 'init', 0)
+    'true' =:= (catch kz_module:is_exported(Module, 'init', 0))
```

Comment:
@jamesaimonetti it shouldn't crash, what am i missing ?

Reply:
i see this in mkbusy test, was that the reason ? why are we trying to convert `cb_d247712c7b03b2114335aa13b95b0849` ?
```
erlang:binary_to_existing_atom(<<"cb_d247712c7b03b2114335aa13b95b0849">>, utf8)
{'EXIT',{{case_clause,[api_resource,<0.2725.0>,13,<0.3106.0>,badarg,[{erlang,binary_to_existing_atom,[<<"cb_d247712c7b03b2114335aa13b95b0849">>,utf8],[]},{kz_term,to_atom,1,[{file,"src/kz_term.erl"},{line,332}]},{kz_module,is_exported,3,[{file,"src/kz_module.erl"},{line,25}]},{api_util,is_cb_module,1,[{file,"src/api_util.erl"},{line,625}]},{api_util,'-parse_path_tokens/3-fun-4-',2,[{file,"src/api_util.erl"},{line,608}]},{lists,splitwith,3,[{file,"lists.erl"},{line,1413}]},{api_util,parse_path_tokens,...},...]]},...}}
```
if this call (with `cb_SOME_ID`)  can be avoided, then maybe the conversion in kz_module can use `kz_term:to_atom/2`
or... do the try catch in kz_module